include/can/core/ids.hpp

```diff
@@ -125,6 +127,13 @@ enum class ErrorCode {
     hardware = 0x5,
 };
 
+/** Error Severity levels */
+enum class ErrorSeverity {
```

Comment:
I think this enum is not being used anywhere yet, could we either remove it for now or add it to the bootloader `id.h` in case we forget to do so later?